src/components/molecules/Visualizer/Engine/ref.ts

```diff
@@ -88,6 +93,11 @@ export type SceneProperty = {
     cameraLimitterTargetWidth?: number;
     cameraLimitterTargetLength?: number;
   };
+  indicator?: {
+    indicator_type: indicatorTypes;
+    indicator_image?: string;
+    img_scale?: number;
```

Comment:
```suggestion
    indicator_image_scale?: number;
```

Rename it because it should be unique among one property schema. `img_scale` is too abstract and it is unclear what the image is. Also manifest on reearth-backend should be updated.